packages/website/package.json

```diff
@@ -8,13 +8,21 @@
     "start": "pnpm next start"
   },
   "dependencies": {
+    "acorn": "8.8.1",
+    "acorn-jsx": "5.3.2",
+    "arweave": "^1.12.2",
+    "fs": "0.0.1-security",
+    "jsdom": "^21.1.0",
+    "net": "^1.0.2",
     "next": "^13.1.4",
     "next-themes": "^0.2.1",
     "nextra": "^2.2.12",
     "nextra-theme-docs": "^2.2.12",
     "react": "^18.2.0",
     "react-dom": "^18.2.0",
-    "sharp": "^0.31.3"
+    "sharp": "^0.31.3",
```

Comment:
do we need all these dependencies and also the newly added "browser" section? i see that we are using the `arweave` dependency, but I wonder if we need these other ones like acorn, fs, jsdom, net, tls, unist-util-remove.